python/paddle/v2/fluid/layers/nn.py

```diff
@@ -789,3 +789,51 @@ def conv2d_transpose(input,
         attrs=op_attr)
 
     return out
+
+
+def reduce_sum(input,
+               dim=None,
+               keep_dim=False,
+               main_program=None,
+               startup_program=None):
+    """
+    Computes the sum of tensor elements over the given dimension. 
+
+    Args:
+        input (Variable): The input variable which is a Tensor or LoDTensor.
+        dim (int|None): The dimension along which the sum is performed. If :attr:`None`, 
+            sum all elements of :attr:`input` and return a Tensor variable with a single 
+            element, otherwise must be in the range :math:`[-rank(input), rank(input))`. 
+            If :math:`dim < 0`, the dimension to reduce is :math:`rank + dim`.
+        keep_dim (bool): Whether to reserve the reduced dimension in the output Tensor.
+            The result tensor will have one fewer dimension than the :attr:`input` unless 
+            :attr:`keep_dim` is true.
+        main_program (Program): The main program.
+        startup_program (Program): The startup program.
+
+    Returns:
+        Variable: The reduced Tensor variable.
+    
+    Examples:
+        .. code-block:: python
+
+            # x is a Tensor variable with following elements:
+            #    [[0.2, 0.3, 0.5, 0.9]
+            #     [0.1, 0.2, 0.6, 0.7]]
+            fluid.layers.reduce_sum(x)  # output tensor: [3.5]
+            fluid.layers.reduce_sum(x, dim=0)  # output tensor: [0.3, 0.5, 1.1, 1.6]
```

Comment:
Please make sure the number of words in a line less than 80.

Reply:
Done.